sdk/servicebus/azure-servicebus/migration_guide.md

```diff
@@ -1,164 +1,444 @@
-and [Async API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/latest/azure.servicebus.aio.html#azure.servicebus.aio.ServiceBusSender)
-* `ServiceBusReceiver` for receiving messages. [Sync API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/latest/azure.servicebus.html#azure.servicebus.ServiceBusReceiver)
-and [Async API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/latest/azure.servicebus.aio.html#azure.servicebus.aio.ServiceBusReceiver)
+In V7:
+```python
+# Authenticate with connection string
+servicebus_client = ServiceBusClient.from_connection_string()
 
-As a user this will be largely transparent to you, as initialization will still occur primarily via the top level ServiceBusClient,
-the primary difference will be that rather than creating a queue_client, for instance, and then a sender off of that, you would simply
-create a servicebus queue sender off of your ServiceBusClient instance via the "get_queue_sender" method.
+# Authenticate with Active Directory:
+from azure.identity import DefaultAzureCredential
+servicebus_client = ServiceBusClient(fully_qualified_namespace, credential=DefaultAzureCredential())
+```
 
-It should also be noted that many of the helper methods that previously existed on the intermediary client (e.g. QueueClient and `peek()`) now
-exist on the receiver (in the case of `peek_messages()`) or sender itself.  This is to better consolidate functionality and align messaging link lifetime
-semantics with the sender or receiver lifetime.
+### Creating sender and receivers
 
-### Client constructors
+- `QueueClient`, `TopicClient` and `SubscriptionClient` have been replaced with methods to create receivers and senders directly from `ServiceBusClient`.
+- To create a receiver for dead-letter sub queue, `get_deadletter_receiver` has been replaced with `get_<queue/subscritpion>_receiver` which takes an optional parameter `sub_queue` for indication.
 
-| In v0.50 | Equivalent in v7 | Sample |
-|---|---|---|
-| `ServiceBusClient.from_connection_string()`    | `ServiceBusClient.from_connection_string()` | [using credential](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus/samples/sync_samples/authenticate_client_connstr.py ) |
-| `QueueClient.from_connection_string()`    | `ServiceBusClient.from_connection_string().get_queue_<sender/receiver>()` | [client initialization](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus/samples/sync_samples/send_queue.py ) |
-| `QueueClient.from_connection_string(idle_timeout=None)` | `ServiceBusClient.from_connection_string.get_queue_receiver(max_wait_time=None)` | [providing a timeout](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus/samples/sync_samples/session_pool_receive.py) |
+In V0.50:
```

Comment:
What is the reason the table format is changed? Also, I think there's a great value in providing the relevant sample in place

Reply:
I totally agree with you that table has the advantage for comparison.

but what I found is that table might be good for comparison of short/brief code snippets, but too crowded for large content.

for example this one:
![image](https://user-images.githubusercontent.com/47871814/103578550-9f011100-4e8b-11eb-9b8d-a1227a607bde.png)

I think the readability decreases as we want to show more contents in a table. one approach might be we further compress the code snippets, but flattening it out also help users who just want to copy and past the code.

---

actually, I think `Message properties migration guide segment` is a perfect case to apply table.